packages/api-client/__tests__/api/getShippingMethods.spec.ts

```diff
@@ -0,0 +1,26 @@
+import getShippingMethods from '../../src/api/getShippingMethods';
+import eligibleShippingMethodsQuery from '../../src/api/getShippingMethods/eligibleShippingMethodsQuery';
+import { Context } from '../../src/types';
+
+describe('[vendure-api-client] getShippingMethods', () => {
```

Comment:
think about different scenarios, like `apollo client function should have been called with passed params`

Reply:
For now, all unit tests are pretty basic stuff. I will refactor them later so that they will serve more purpose.